frontend/packages/topology/src/components/page/TopologyView.tsx

```diff
@@ -160,16 +161,26 @@ export const ConnectedTopologyView: React.FC<ComponentProps> = ({
   const searchParams = queryParams.get('searchQuery');
   const fileTypes = supportedFileExtensions.map((ex) => `.${ex}`).toString();
 
-  const onSelect = React.useCallback((entity?: GraphElement) => {
-    // set empty selection when selecting the graph
-    const selEntity = isGraph(entity) ? undefined : entity;
-    setSelectedEntity(selEntity);
-    if (!selEntity) {
-      removeQueryArgument('selectId');
-    } else {
-      setQueryArgument('selectId', selEntity.getId());
-    }
-  }, []);
+  const onSelect = React.useCallback(
+    (entity?: GraphElement) => {
+      if (!entity) {
+        sessionStorage.removeItem(LAST_TOPOLOGY_OVERVIEW_OPEN_STORAGE_KEY);
+      }
+      // set empty selection when selecting the graph
+      const selEntity = isGraph(entity) ? undefined : entity;
+      setSelectedEntity(selEntity);
+      if (!selEntity) {
+        removeQueryArgument('selectId');
+      } else {
+        setQueryArgument('selectId', selEntity.getId());
+        sessionStorage.setItem(
+          LAST_TOPOLOGY_OVERVIEW_OPEN_STORAGE_KEY,
+          JSON.stringify({ [namespace]: selEntity.getId() }),
```

Comment:
This removes the selected entry from another project which IMHO doesn't align with the AC "per project".

I would recommend to parse and spread the old values from sessionStorage

Reply:
PM requirement for this [story](https://issues.redhat.com/browse/ODC-6053) is to just remember the node selection for the last active namespace. cc: @serenamarie125

Reply:
AC 1 sounds like we should save it for all namespaces. AC 2 is not added by PM. So let see if we get some feedback from Serena here. :)

@serenamarie125 @beaumorley I'm talking about this case:

* User selects a Deployment in project1
* User switch to project2 to check something, selects another Deployment
* User switch back to project1. Should we automatically select the first Deployment again?

The change will be straightforward, don't worry about that.